presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -299,11 +297,12 @@ private static URI getQueuedUri(QueryId queryId, Slug slug, long token, UriInfo
     }
 
     /**
-     * Instantiates an object that contains all information about query metadata.
+     * Builds the object that holds all metadata about query result.
```

Comment:
```suggestion
     * Instantiates the object that holds all metadata about a query result.
```

I think it's better to use the word "Instantiate" instead of "Build", to avoid confusion with the Builder Design Pattern

Reply:
aggred